cmd/repo-init/main.go

```diff
@@ -581,6 +581,10 @@ Ensure that webhooks are set up for Prow to watch GitHub state.`)
 		Repos:            []string{orgRepo},
 		ReviewActsAsLgtm: true,
 	})
+	pluginConfig.Triggers = append(pluginConfig.Triggers, plugins.Trigger{
+		Repos:       []string{orgRepo},
+		TrustedApps: []string{},
+	})
```

Comment:
I don't see why this would be necessary - why generate what is basically an empty config?